src/app/Routing.tsx

```diff
@@ -20,6 +22,9 @@ const Routing: React.FC = () => {
       <Route path="/login" element={<LoginPage />} />
       <Route path="/register" element={<LoginPage />} />
       <Route path="/404" element={<NoMatch />} />
+
+      <Route path="/start" element={<StartPage />} />
+      <Route path="/dodatkowe" element={<AdditionalPage />} />
```

Comment:
We should discuss path naming whether it should be in english or polish.

Reply:
agree, lets save it for monday